### PR TITLE
AppVeyor: Check if requirements for codesigning are met

### PR DESCRIPTION
**Pull request by Holzhaus**
Attempt to restore WIndows builds on private forks.

### CODE REVIEW

**Comment by Holzhaus** [comment]
Oops, that won't work.

**Review by uklotzde — APPROVED** [review]
Works as intended. Thank you! Merge?

File: appveyor.yml [hunk 1]
```diff
@@ -191,10 +191,11 @@ for:
     - python -m pip install git+https://github.com/frerich/clcache.git
     # Decrypt the code signing certificate.
     - ps: iex ((New-Object Net.WebClient).DownloadString('https://raw.githubusercontent.com/appveyor/secure-file/master/install.ps1'))
-    - IF [%APPVEYOR_PULL_REQUEST_NUMBER%] == [] (
+    - If [%APPVEYOR_PULL_REQUEST_NUMBER%] == []
+      If "%APPVEYOR_PROJECT_NAME%" == "mixxxdj" (
         appveyor-tools\secure-file -decrypt %CODESIGN_CERTIFICATE_PATH%.enc -secret %CODESIGN_CERTIFICATE_SECURE_FILE_SECRET% -salt %CODESIGN_CERTIFICATE_SECURE_FILE_SALT%
       ) else (
-        echo "Skipping certificate decryption because we are in a pull request."
+        echo "Skipping certificate decryption because we are in a pull request or not in the mixxxdj project."
```
Comment (uklotzde) [inline appveyor.yml:198]: mixxxdj is not the project but the account.

File: appveyor.yml [hunk 2]
```diff
@@ -248,17 +249,19 @@ for:
       ..
     - cmake --build .
     - cmake --build . --target install
-    - IF [%APPVEYOR_PULL_REQUEST_NUMBER%] == [] (
+    - If [%APPVEYOR_PULL_REQUEST_NUMBER%] == []
+      If "%APPVEYOR_PROJECT_NAME%" == "mixxxdj" (
         signtool sign /f ..\%CODESIGN_CERTIFICATE_PATH% /p %CODESIGN_CERTIFICATE_PASSWORD% *.exe
       ) else (
-        echo "Skipping codesigning *.exe because we are in a pull request."
+        echo "Skipping certificate decryption because we are in a pull request or not in the mixxxdj project."
```
Comment (uklotzde) [inline appveyor.yml:256]: project -> account

File: appveyor.yml [hunk 3]
```diff
@@ -248,17 +249,19 @@ for:
...
       )
     # Generate WIX installer
     - cpack -G "WIX"
-    - IF [%APPVEYOR_PULL_REQUEST_NUMBER%] == [] (
+    - If [%APPVEYOR_PULL_REQUEST_NUMBER%] == []
+      If "%APPVEYOR_PROJECT_NAME%" == "mixxxdj" (
         signtool sign /f ..\%CODESIGN_CERTIFICATE_PATH% /p %CODESIGN_CERTIFICATE_PASSWORD% *.msi
       ) else (
-        echo "Skipping codesigning *.msi because we are in a pull request."
+        echo "Skipping certificate decryption because we are in a pull request or not in the mixxxdj project."
```
Comment (uklotzde) [inline appveyor.yml:264]: project -> account

**Review by uklotzde — APPROVED** [review]
LGTM. Thank you!

**Pull request merged**